public/src/components/student/SubmissionTypeIcon.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react'
 
 interface SubmissionTypeIconProps {
-  solo: boolean
+    solo: boolean
 }
 
 const SubmissionTypeIcon: React.FC<SubmissionTypeIconProps> = ({ solo }) => {
     const indicator = solo ? "fa-user" : "fa-users"
     return (
         <i
             className={`fa ${indicator} submission-icon`}
-            title={solo ? "Solo submission" : "Group submission"}
```

Comment:
I like this variant better. It is easier to read and find if searching in the codebase for `Group submission` or `Solo submission`